reana_commons/workspaces.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA-Commons workspaces util."""
+
+from reana_commons.errors import REANAValidationError
+from reana_commons.config import WORKSPACE_PATHS
+
+
+def validate_workspace(workspace_option):
+    """Validate and return workspace.
+
+    :param workspace_option: A string of the workspace.
+    :returns: A string of the validated workspace.
+    """
+    if workspace_option:
+        if workspace_option not in WORKSPACE_PATHS:
```

Comment:
We need to take into account that one of the `WORKSPACE_PATHS` could be for example `/eos`,
but the user defined workspace would be `/eos/user/a/amecioni/workflows/my_super_workflow.` So direct comparison would not allow this workspace definition even though it's perfectly valid. I guess for now we could iterate through the `WORKSPACE_PATHS` list and check if `workspace_option` `startswith()` at least one path?

Later some more complex validation could be done (better in a separate PR in the future) to catch all the possible ways to abuse the system, like to catch "../", etc. (thanks @mvidalgarcia for noticing this possibility)